storybook/.storybook/main.ts

```diff
@@ -6,6 +6,7 @@ const config: StorybookConfig = {
     builder: 'webpack4',
   },
   features: {
+    postcss: false,
```

Comment:
Hides a warning in the console about postcss.
I might turn it back on in a future PR to add a Tailwind styled story.